doc/releases/release-notes-template.rst

```diff
@@ -0,0 +1,269 @@
+:orphan:
+
+.. _zephyr_X.Y:
+
+Zephyr X.Y.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version X.Y.0.
+
+.. This section is to be populated by the release manager
+
+Major features and improvements in this release include:
+
+*
+*
+
+API Changes
+***********
+
+.. Please document all API additions, removals, deprecation and changes here and
+   not in the individual subsections
+
+New APIs
+========
+
+Removed APIs
+============
+
+Deprecated APIs
+===============
+
+Changed APIs
+============
+
+.. Includes both stable and unstable/experimental API changes
+
+Security Vulnerabilities
+************************
+
+Detailed changelog
+******************
+
+Kernel
+======
+
+Architectures
+=============
+
+* ARC
+
+* ARM
+
+* ARM64
+
+* MIPS
+
+* NIOS2
+
+* POSIX
+
+* RISC-V
+
+* SPARC
+
+* x86
+
+* Xtensa
+
+Bluetooth
+=========
+
+* Audio
+
+* Direction Finding
+
+* Host
+
+* Mesh
+
+* Controller
+
+* HCI Driver
+
+Boards & SoC Support
+====================
+
+* SoC series
+
+* ARC boards
+
+* ARM boards
+
+* ARM64 boards
+
+* MIPS boards
+
+* NIOS2 boards
+
+* POSIX boards
+
+* RISC-V boards
+
+* SPARC boards
+
+* x86 boards
+
+* Xtensa boards
+
+* Other boards
+
+* Shields:
+
+Build and configuration systems
+===============================
+
+Devicetree
+==========
+
+Device Management
+=================
+
+Documentation
+=============
+
+Drivers
+=======
+
+* ADC
+
+* Battery-backed RAM
+
+* CAN
+
+* Clock control
+
+* Counter
+
+* Crypto
+
+* DAC
+
+* DFU
+
+* Disk
+
+* Display
+
+* DMA
+
+* EEPROM
+
+* Entropy
+
+* ESPI
+
+* Ethernet
+
+* Flash
+
+* FPGA
+
+* Fuel Gauge
+
+* GPIO
+
+* hwinfo
+
+* I2C
+
+* I2S
+
+* I3C
+
+* IEEE 802.15.4
+
+* Interrupt Controller
+
+* IPM
+
+* KSCAN
+
+* LED
+
+* MBOX
+
+* MEMC
+
+* PCIE
+
+* PECI
+
+* Pin control
+
+* PWM
+
+* Power domain
+
+* Regulators
+
+* Reset
+
+* RTC
+
+* SDHC
+
+* Serial
+
+* SPI
+
+* Timer
+
+* USB
+
+* W1
+
+* Watchdog
+
+* WiFi
+
+Logging
+=======
+
+Modules
+=======
+
+HALs
+----
+
+mbedTLS
+-------
+
+MCUboot
+-------
+
+Trusted Firmware-M
+------------------
+
+zcbor
+-----
+
+Networking
+==========
+
+Other Subsystems and Libraries
+==============================
+
+Sensors
+=======
+
+Shell
+=====
+
+Storage and file systems
+=========================
+
+Tests and Samples
+=================
+
+USB
+===
+
+Issues
+******
+
+Known Issues
+============
+
+Addressed issues
+================
```

Comment:
These two were gone in 3.4, the list was super noisy anyway. How about either we drop them entirely, or maybe we just keep Known Issues and list the currently opened high and mid priority there? That should hopefully keep it short enough that one may actually read it and noise free since that list is maintained.